### PR TITLE
Adding v2 of ssa___rare_..._lolbas

### PR DESCRIPTION
**Pull request by peter-cg**

### CODE REVIEW

**Comment by peter-cg** [comment]
I read somewhere that the extra CI validation won't happen unless the branch starts with "ssa" so I'm trying that..

File: detections/endpoint/ssa___rare_parent_process_relationship_lolbas.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  "_time"), "string", null)) | eval parent_process=lower(ucast(map_get(input_event,
+  "parent_process_name"), "string", null)), parent_process_name=mvindex(split(parent_process,
+  "\\"), -1), process_name=lower(ucast(map_get(input_event, "process_name"), "string",
+  null)), cmd_line=ucast(map_get(input_event, "process"), "string",
+  null), dest_user_id=ucast(map_get(input_event, "dest_user_id"), "string", null),
+  dest_device_id=ucast(map_get(input_event, "dest_device_id"), "string", null)
+  | where parent_process_name!=null
+  | select parent_process_name, process_name, cmd_line, timestamp, dest_device_id, dest_user_id
+  | conditional_anomaly conditional="parent_process_name" target="process_name"
+  | where (process_name="powershell.exe" OR process_name="regsvcs.exe"
```
Comment (jzsplunk) [inline detections/endpoint/ssa___rare_parent_process_relationship_lolbas.yml:22]: @peter-cg do we end up reusing this same list of process names in many detections would it make sense to create a macro for this whitelist at some point?

**Review by jzsplunk — COMMENTED** [review]
LGTM

File: detections/endpoint/ssa___rare_parent_process_relationship_lolbas.yml (hunk 1 above)
Comment (peter-cg) [reply detections/endpoint/ssa___rare_parent_process_relationship_lolbas.yml:22]: It should be in a lookup when that's possible, yeah.

**Comment by P4T12ICK** [comment]
Close it because I merged it with the other PR.

**Pull request closed without merging**